overlord/snapstate/snapmgr.go

```diff
@@ -69,6 +69,7 @@ const defaultRefreshSchedule = "00:00-04:59/5:00-10:59/11:00-16:59/17:00-23:59"
 
 // overridden in the tests
 var errtrackerReport = errtracker.Report
+var miscRefreshDelay = 24 * time.Hour
```

Comment:
this probably merits its own comment/explanation,  also perhaps  s/misc/catalogs/ or something along those lines?